### PR TITLE
Feat #31 add endpoint for subjects

### PR DESCRIPTION
**Pull request by MuradAbbaszade**
Endpoint added

### CODE REVIEW

File: docker-compose.yml
Comment (hamidsultanzadeh) [inline docker-compose.yml:1]: keep this file unchanged

**Comment by hamidsultanzadeh** [comment]
please, look at the comments and fix them

use business service for inserting process.
create new method for checking existence of language, like isExist (look at UserService)

**Review by hamidsultanzadeh — CHANGES_REQUESTED** [review]

**Review by aytiqaqash — CHANGES_REQUESTED** [review]
Based on my knowledge, the best practices of REST API verbs should not be in the route. Instead of "/add" suggesting name "/subject". So it will be POST a subject.

**Review by aytiqaqash — APPROVED** [review]
LGTM

**Review by hamidsultanzadeh — APPROVED** [review]

**Pull request merged**